node/declaration.ts

```diff
@@ -75,4 +77,35 @@ export const getDeclarationsFromConfig = (
   return declarations
 }
 
+// Validates and normalizes a pattern so that it's a valid regular expression
+// in Go, which is the engine used by our edge nodes.
+export const parsePattern = (pattern: string) => {
+  // Escaping forward slashes with back slashes.
+  const normalizedPattern = pattern.replace(/\//g, '\\/')
+  const regex = regexpAST.transform(`/${normalizedPattern}/`, {
+    Assertion(path) {
+      // Lookaheads are not supported. If we find one, throw an error.
+      if (path.node.kind === 'Lookahead') {
+        throw new Error('Regular expressions with negative lookaheads are not supported')
```

Comment:
It's not just negative lookaheads that aren't supported: positive lookaheads aren't either.

Reply:
Ah-ha! The comment was correct but the error message was not. Fixed in https://github.com/netlify/edge-bundler/pull/298/commits/2efa3f819951ae2a7c1ea2ec28bbb3a85cd43145.